Source/WebInspectorUI/UserInterface/Views/ConsoleGroup.js

```diff
@@ -79,6 +81,36 @@ WI.ConsoleGroup = class ConsoleGroup extends WI.Object
     {
         this._messagesElement.appendChild(messageOrGroupElement);
     }
+    
+    // FIXME: <https://webkit.org/b/264489> We should merge this logic with `WI.ConsoleMessageView.prototype.render`
+    setMessageLevel(messageLevel)
+    {
+        if (!this._element)
+            return;
+  
+        switch (messageLevel) {
```

Comment:
do we need to cache `messageLevel` in case this is called _before_ `this._element` is initialized (i.e. this would automatically get called after `this._element` is created)?  or is it always expected that this will be called after (which is what it seems like this PR does)?

if the latter, we should replace the early return `if` above to just be a `console.assert(this._element);`

Reply:
It is designed to be called after `this._element` is initialized.